controllers/homeController.js

```diff
@@ -76,6 +79,66 @@ var findCommonChampions = function(championsA, championsB) {
     return commonChampions
 }
 
+var compareCommonChampions = function(commonChampions) {
+    var ret = {
+        // overall result for all the champions.
+        compare: 0,
+        // comparison for each champion.
+        // array will consist of json like {id: champion id, result: return value from compareChampionStats}
+        championCompareList: []
+    };
+
+    commonChampions.forEach(([championA, championB]) => {
```

Comment:
nit- how would you feel if we made this a tad bit more functional? rather than foreach use map and return {id:... , result...} so commonChampions will map to championCompareList and you can have a variable compare that you append to inside the method. Then at the very end you build your ret object with championCompareList and compare variable.